common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -176,29 +176,6 @@ def _auto_auth(self, username, email, staff, enrollment_mode="honor"):
         AutoAuthPage(self.browser, username=username, email=email,
                      course_id=self.course_id, staff=staff, enrollment_mode=enrollment_mode).visit()
 
-    def _create_a_proctored_exam_and_attempt(self):
-        """
-        Creates a proctored exam and makes the student attempt it so that
-        the associated allowance and attempts are visible on the Instructor Dashboard.
-        """
-        # Visit the course outline page in studio
-        LogoutPage(self.browser).visit()
-        self._auto_auth("STAFF_TESTER", "staff101@example.com", True)
-        self.course_outline.visit()
-
-        #open the exam settings to make it a proctored exam.
-        self.course_outline.open_exam_settings_dialog()
-        self.course_outline.make_exam_proctored()
-        time.sleep(2)  # Wait for 2 seconds to save the settings.
-
-        # login as a verified student and visit the courseware.
-        LogoutPage(self.browser).visit()
-        self._auto_auth(self.USERNAME, self.EMAIL, False, enrollment_mode="verified")
-        self.courseware_page.visit()
-
-        # Start the proctored exam.
-        self.courseware_page.start_proctored_exam()
-
```

Comment:
@muhhshoaib even though we might not be calling this helper method, it seems like it'd be useful to keep around for future use. Is there any particular reason why we should delete it except for remove uncalled code?